app/charts/shared/legend-color.tsx

```diff
@@ -182,6 +183,7 @@ export const LegendColor = memo(function LegendColor(props: LegendColorProps) {
     symbol,
     getLegendItemDimension,
     interactive,
+    visible = true,
```

Comment:
@noahonyejese I think I wouldn't use a default `true` here – old charts shouldn't be changed to show legend titles IMO, as in case they are embedded somewhere, that would result in layout shifts and possibly scrollable or cut iframes 🥹